src/js/index.js

```diff
@@ -0,0 +1,191 @@
+// 현재 메뉴 상태를 관리하는 변수
+const state = MenuStatus.loadOrCeateNewMenuStatus();
+
+const inputTag = document.getElementById("espresso-menu-name");
+const menuForm = document.getElementById("espresso-menu-form");;
+const menuList = document.getElementById("espresso-menu-list");
+const menuCount = document.querySelector(".menu-count");
+const menuHeader = document.querySelector(".heading h2");
+const menuTypeSelectButtonWarpper = document.querySelector("header nav");
+
+menuForm.addEventListener("submit", onSubmitForm);
+menuList.addEventListener("click", onMenuClicked);
+menuTypeSelectButtonWarpper.addEventListener("click", onMenuTypeClicked);
+document.addEventListener("DOMContentLoaded", renderMenu);
+
+function onSubmitForm(e) {
+	e.preventDefault();
+	const inputString = String(inputTag.value).trim();
```

Comment:
`nputTag.value` 는 원래 `String`인데 `trim`을 사용하시기 위해 `String` 으로 선언 하신 걸까요~? 
`optional` 을 이용해서 좀 더 심플하게 작성 할 수 있을 것 같아요.

```suggestion
	const inputString = inputTag.value?.trim();
```

 value가 undefined 이거나 null 인 경우에는 trim 메소드가 실행되지 않으니 적지만 성능도 좀 더 좋아지겠네요!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
오 넵넵

> nputTag.value 는 원래 �String인데 trim을 사용하시기 위해 String 으로 선언 하신 걸까요~?
> ㄴ 넵넵 맞습니다! undefind 값 올까봐 String()으로 한번 묶어줬습니다!

그렇네요 optional 적용하면 더 깔끔하게 코드 작성 가능하네요!
감사합니다! :)